core/internal/testdata/flux_monitor_job.json

```diff
@@ -1,4 +1,12 @@
 {
+  "comment_comments": [
```

Comment:
I think these comments would be more discoverable in a package than in a test fixture.

Reply:
I can remove them, but FWIW, the job specs were one of the first places I looked when I was figuring chainlink out, and they were confusing.